Intersect (Core)/Network/Packets/Server/ItemProperties.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using Intersect.Enums;
+using Intersect.GameObjects;
+using Intersect.Utilities;
+using MessagePack;
+
+namespace Intersect.Network.Packets.Server
+{
+    [MessagePackObject]
+    public partial class ItemProperties
+    {
+        public ItemProperties()
+        {
+        }
+
+        public ItemProperties(Guid itemBaseId)
+        {
+            var descriptor = ItemBase.Get(itemBaseId);
+            if (descriptor == null)
+            {
+                return;
+            }
+
+            if (descriptor.ItemType != ItemTypes.Equipment)
+            {
+                return;
+            }
+
+            for (var i = 0; i < (int)Stats.StatCount; i++)
+            {
+                // TODO: What the fuck?
+                RandomStats[i] = Randomization.Next(-descriptor.StatGrowth, descriptor.StatGrowth + 1);
+            }
```

Comment:
Add a method in `Randomization` with the signature `public static void NextValues(int[] buffer)` and it should fill the buffer with random integers.

In the `Item` implementation on the server it should fill StatModifiers using `Randomization.NextValues(properties.StatModifiers)` according to the equipment logic and the stat randomizer boolean.